src/routes/task.ts

```diff
@@ -1,104 +1,21 @@
 import express from 'express';
 import { create, getAll, getId, remove, update } from '../controllers/taskController';
+import { requireAuthAPI } from '../middleware/authMiddleware';
 
-const taskRouter = express.Router();
+const TaskRouter = express.Router();
```

Comment:
You Capitalized My TaskRouter?

Reply:
Ummmmmmm..... idk I think I was still in C++ mode?